hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@hyi - the gist of what I told @pkdash was that at the time a resource is published, it is published by the "publisher".  This is a matter of fact.  It simply records what occurred at that moment in time.  As we can't change the past, it makes sense to set the "publisher" and not update it.